website_product_configurator/README.md

```diff
@@ -0,0 +1,3 @@
+#Odoo Product Configurator
+
+This module facilitates to configure product on website.
```

Comment:
Could you replace this with proper "readme section files" under a `readme` directory? 🙏🏻